pkg/asset/installconfig/pullsecret.go

```diff
@@ -24,7 +24,7 @@ func (a *pullSecret) Generate(asset.Parents) error {
 		{
 			Prompt: &survey.Input{
 				Message: "Pull Secret",
-				Help:    "The container registry pull secret for this cluster, as a single line of JSON (e.g. {\"auths\": {...}}).\n\nYou can get this secret from https://try.openshift.com",
+				Help:    "The container registry pull secret for this cluster, as a single line of JSON (e.g. {\"auths\": {...}}).\n\nYou can get this secret from https://cloud.openshift.com/clusters/install#pull-secret",
```

Comment:
The `#pull-secret` anchor exists on the page, but doesn't really work — it's lost somewhere in auth redirects (cloud.openshift.com goes to developers.redhat.com, which, if logged in immediately redirects back to `https://cloud.openshift.com/clusters/install#state=ebc6d391-...&session_state=1de96462-...` — anyway the fragment is lost).

Is this important, do you want to get that fixed?  (I'm not sure how exactly, but will at least track it on our side...)

Reply:
> The `#pull-secret` anchor exists on the page, but doesn't really work — it's lost somewhere in auth redirects...

I'd pointed that out to @kbsing, but yeah, if you want to file an issue to track it, that would be great :).